src/main/java/com/gitrekt/resort/controller/ReportsHomeScreenController.java

```diff
@@ -51,4 +54,14 @@ public void onBookingPercentagesReportButtonClicked() {
         // TODO
     }
     
+    //Opens FeedbackReportScreen.fxml when button Feedback Report button is clicked
+     public void onFeedbackReportButtonClicked() throws IOException{
+        Stage mainStage = (Stage) feedbackReportButton.getScene().getWindow();
+        Parent staffHomeScreenRoot = FXMLLoader.load(
+            getClass().getResource("/fxml/FeedbackReportScreen.fxml")
+        );
+        Scene staffHomeScreen = new Scene(staffHomeScreenRoot);
+        mainStage.centerOnScreen();
```

Comment:
Same as previous; you must center the screen after you set the scene, not before. This logic will all possibly changed later with some kind of screen managing class (hopefully, if I can figure out how to implement one that isn't terrible), so it's not a huge deal right now.